test/test_um2netcdf.py

```diff
@@ -116,6 +118,14 @@ def coord(self, _name):
             msg = f"{self.__class__}[{self.var_name}]: lacks coord for '{_name}'"
             raise CoordinateNotFoundError(msg)
 
+    def update_coords(self, coords):
+        # Mimic a coordinate dictionary keys for iris coordinate names. This
```

Comment:
This is where the `Mock` class was useful, it could be passed an object & dynamically make an interface from that.

I can prefix helper funcs with `aux_` & add comments to clarify non-iris interface elements. Would that help?

Reply:
I think that would be helpful!

Reply:
The fix is in d3164e3d.